cmd/wallet/address.go

```diff
@@ -3,206 +3,233 @@ package main
 import (
 	"fmt"
 
-	cli "github.com/jawher/mow.cli"
 	"github.com/pactus-project/pactus/cmd"
 	"github.com/pactus-project/pactus/crypto/bls"
 	"github.com/pactus-project/pactus/util"
+	"github.com/spf13/cobra"
 )
 
+var addrCmd = &cobra.Command{
```

Comment:
You can move this car inside the `buildAllAddrCmd` (without var)

Reply:
just this?
or I should move all Cmd to their build func?